tests/integration/suite/daprd/jobs/http/basic.go

```diff
@@ -84,7 +84,8 @@ func (b *basic) Run(t *testing.T, ctx context.Context) {
 		for _, body := range []string{
 			`{"schedule": "@daily"}`,
 			`{"schedule": "@daily", "repeats": 3, "due_time": "10s", "ttl": "11s"}`,
-			`{"schedule": "@daily", "repeats": 3, "due_time": "10s", "ttl": "11s", "data": {"@type": "type.googleapis.com/google.protobuf.StringValue", "value": "Hello, World!"}}`,
+			`{"schedule": "@daily", "repeats": 3, "due_time": "10s", "ttl": "11s", "data": "{\"@type\": \"type.googleapis.com/google.protobuf.StringValue\", \"value\": \"Hello, World!\"}"}`,
```

Comment:
I thought we agreed not to have this format anymore. I am OK for now, but I vote for this to be removed for HTTP in 1.15.

Reply:
Yes. We don't ingest data payloads as proto objects and instead as JSON strings. as agreed.

It's still valid JSON though, and will be ingested as such. Please see the addition of quotations, and the next test case.